src/app/me/sources.ts

```diff
@@ -0,0 +1,16 @@
+import { DataSourceResponse } from '@/app/application/services/data-source/DataSourcePool'
+
+type Closeable = { close: () => void }
+
+export type MeSource = DataSourceResponse<{
+  pageSize: number
+}>
+
+export const sources = () => {
+  return {
+    '/me': async (_params = {}, source: Closeable) => {
+      source.close()
+      return { pageSize: 50 }
```

Comment:
This will eventually come from localStorage, and/or default to `50`. We'll only use the default `50` in one place, so I left the CONSTANT behind.